charts/zeppelin/values.yaml

```diff
@@ -0,0 +1,271 @@
+    # - secretName: zeppelin-server-tls
+    #   hosts:
+    #   - local.zeppelin-project.org
+
+interpreter:
+  image:
+    # -- Zeppelin interpreter image registry.
+    # @default -- If not set, `global.image.registry` will be used.
+    registry: ""
+    # -- Zeppelin interpreter image repository.
+    repository: apache/zeppelin-interpreter
+    # -- Zeppelin interpreter image tag.
+    # @default -- If not set, the chart appVersion will be used.
+    tag: ""
+    # -- Zeppelin interpreter image pull policy.
+    pullPolicy: IfNotPresent
+    # -- Zeppelin interpreter image pull secrets for private image registry.
+    pullSecrets: []
+    # - name: <secret-name>
+
+  spark:
+    image:
+      # -- Spark image registry.
+      # @default -- If not set, `global.image.registry` will be used.
+      registry: ""
+      # -- Spark image repository.
+      repository: spark
+      # -- Spark image tag.
+      tag: 3.5.3
+    
+    # -- Spark configurations.
+    conf:
+      spark.kubernetes.driver.podTemplateFile: file:///opt/spark/conf/driver-pod-template.yaml
+      spark.kubernetes.executor.podTemplateFile: file:///opt/spark/conf/executor-pod-template.yaml
+
+    # -- Extra labels for Spark pods.
+    labels: {}
+
+    # -- Extra annotations for Spark pods.
+    annotations: {}
+
+    # -- Volumes for Spark pods.
+    volumes: []
+
+    # -- Environment variables for Spark containers.
+    env: []
+
+    # -- Environment variable sources for Spark containers.
+    envFrom: []
+
+    # -- Volume mounts for Spark containers.
+    volumeMounts: []
+
+  # -- Extra labels for Zeppelin interpreter pods.
+  labels: {}
+    # key1: value1
+    # key2: value2
+
+  # -- Extra annotations for Zeppelin interpreter pods.
+  annotations: {}
+    # key1: value1
+    # key2: value2
+
+  # -- Volumes for Zeppelin interpreter pods.
+  volumes: []
+
+  # -- Node selector for Zeppelin interpreter pods.
+  nodeSelector: {}
+
+  # -- Affinity for Zeppelin interpreter pods.
+  affinity: {}
+
+  # -- List of node taints to tolerate for Zeppelin interpreter pods.
+  tolerations: []
+
+  # -- Priority class for Zeppelin interpreter pods.
+  priorityClassName: ""
+
+  # -- Security context for Zeppelin interpreter pods.
+  podSecurityContext: {}
+
+  # -- Environment variables for Zeppelin interpreter containers.
+  env:
+  - name: ZEPPELIN_HOME
+    value: /opt/zeppelin
+  - name: SPARK_HOME
+    value: /opt/spark
+  - name: SPARK_CONF_DIR
+    value: /opt/spark/conf
+
+  # -- Environment variable sources for Zeppelin interpreter containers.
+  envFrom: []
+
+  # -- Volume mounts for Zeppelin interpreter containers.
+  volumeMounts: []
+
+  # -- Resource requests and limits for Zeppelin interpreter containers.
+  resources: {}
+    # limits:
+    #   cpu: 100m
+    #   memory: 300Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 300Mi
+
+  # -- Security context for Zeppelin interpreter containers.
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 1000
```

Comment:
Please do not enter a fixed UID. Openshift, for example, uses a random UID by default.

Reply:
I have not tried whether it can work with a random UID. The uid `1000` comes from the zeppelin server Dockerfile, ref: https://github.com/apache/zeppelin/blob/b6e40d4224a274d96834e75453724451f9e557fe/scripts/docker/zeppelin-server/Dockerfile#L79-L85.